private/gen/proto/apiclient/apiclient.pb.go

```diff
@@ -20,6 +20,27 @@ import (
 	registryv1alpha1apiclient "github.com/bufbuild/buf/private/gen/proto/apiclient/buf/alpha/registry/v1alpha1/registryv1alpha1apiclient"
 )
 
+const (
+	// ProviderSchemeHTTP represents a provider scheme of http
+	ProviderSchemeHTTP ProviderScheme = iota + 1
+	// ProviderSchemeHTTPS represents a provider scheme of https
+	ProviderSchemeHTTPS
+)
+
+// ProviderScheme represents a scheme that will be used by all clients returned from a given provider
+type ProviderScheme int
```

Comment:
Why are we generating this?

I also wonder why we need it, probably worth hopping on a Zoom

Reply:
Talked through this, we should definitely kill this heh (or move it to a non-generated place), I think `WithAddressMapper` takes care of the concern though